src/util/Whamm.v3

```diff
@@ -281,93 +299,125 @@ class WhammVarBinder(wi: WhammInstance, err: ErrorGen) {
 		match (result) {
 			Value(vals) => {
 				if (vals.length != 1) {
-					err.at(cv.parser.decoder).WhammProbeError("whamm probe", "incorrect function return arity");
+					err.at(cv.parser.decoder).WhammProbeError("@bind", "incorrect function return arity");
```

Comment:
Can you update the error messages so that they explain what was expected and what was encountered?